content/manuals/build-cloud/release-notes.md

```diff
@@ -0,0 +1,16 @@
+---
+description: Learn about the latest features of Docker Build Cloud
+keywords: docker build cloud, release notes, changelog, features, changes, delta, new, releases
+title: Docker Build Cloud release notes
+linkTitle: Release notes
+tags: [Release notes]
+---
+
+This page contains information about the new features, improvements, known
+issues, and bug fixes in Docker Build Cloud releases. 
+
+## 2025-02-24
+
+### New
+
+Added a new **Build settings** page where you can configure disk allocation, private resource access, and firewall settings for your cloud builders in your organization. These configurations help optimize storage, enable access to private registries, and secure outbound network traffic.
```

Comment:
nit: should we have a link to the builder settings documentation, as soon as it's merged ?